tests/common.rs

```diff
@@ -39,6 +41,7 @@ pub fn build_hermit_bin(kernel: impl AsRef<Path>) -> PathBuf {
 /// Small wrapper around [`Uhyve::run`] with default parameters for a small and
 /// simple Uhyve vm
 pub fn run_simple_vm(kernel_path: PathBuf) {
+	println!("Launching kernel in path: {}\n", kernel_path.display());
```

Comment:
`println!("Launching kernel {}", kernel_path.display());`

Reply:
Just to confirm: Given that we mention the kernel earlier (referring to `gdb`, `serial`, etc.)...

https://github.com/hermit-os/uhyve/pull/759/files#diff-e18d1086c27bd660063a2415f5ee672715d7717296abd9cbb6441043c6e289fcR15

... is conflating the kernel's path / location and the kernel's "name" a good idea?

Reply:
It should be clear from the context. But you can remove the "name".

Reply:
your call, pushing a change soon

Reply:
`println!("Launching kernel {}", kernel_path.display());`